sretoolbox/container/image.py

```diff
@@ -243,7 +260,8 @@ def _parse_image_url(image_url):
             else:
                 image_url_struct['repository'] = None
 
-        if image_url_struct.get('tag') is None:
+        # By-digest URIs don't use tags; but otherwise default to `latest` if absent
+        if image_url_struct.get('tag') is None and not image_url_struct.get('by_digest'):
```

Comment:
`... and image_url_struct.get('by_digest') is None`, for consistency & correctness

Reply:
Roger that